com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Settings/FrameSettings.cs

```diff
@@ -350,7 +350,9 @@ public enum FrameSettingsField
     [DebuggerDisplay("{mask.humanizedData}")]
     public struct FrameSettingsOverrideMask
     {
-        /// <summary>Mask of overridden values.</summary>
+        /// <summary>Gets the underlying BitArray HDRP uses to store the override mask and thus specific which field is overridden or not.
+        /// Note: BitArray128 is implements IBitArray and therefore has the scripting API described below. It is recomended to use the interface as the exact BitArray con evolve from one version of the package to another as the we need more capacity here.
```

Comment:
BitArray128 implements IBitArray and therefore includes the IBitArray scripting API. Be sure to use the interface as the exact BitArray type may evolve from one version of the package to another as HDRP requires more capacity.